docs/section21/Rule21-4.md

```diff
@@ -0,0 +1,45 @@
+
+# Boiler - Rule 21-4  
+
+**Rule ID:** 21-4  
+**Rule Description:** When baseline building does not use purchased heat, baseline systems 1,5,7,11,12 shall be modeled with natural draft boilers.  
+**Rule Assertion:** B-RMR  
+**Appendix G Section:** Section 21 Boiler  
+**Appendix G Section Reference:** Section G3.1.3.2 Building System-Specific Modeling Requirements for the Baseline model  
+
+**Applicability:** All required data elements exist for B_RMR  
+**Applicability Checks:**  
+
+1. Baseline number of HW Loops > 0 and number of boilers in HW loop > 0.  
+
+**Manual Check:** None  
+**Evaluation Context:** Building  
+**Data Lookup:** None  
+**Function Call:** None  
+
+## Rule Logic:  
+
+- For each conditioning component in B_RMR: `for conditioning_component_b in B_RMR.ASHRAE229.conditioning_components:`
+
```

Comment:
@xingcx i dont quite follow what 'conditioning_components' is. 


@mkarpman  - i am not sure how this test would work. My suggestion is that to verify that the baseline has a boiler, we would have to reference a function we haven't written yet to identify the baseline system type(s) in the B-RMR.
The function would be able to go through each zone and identify the baseline system type for that zone.
so if the outcome of that function identifies system types 1,5,7,11 or 12 for any zone, we need to verify that the heating source is a natural draft boiler for that zone.

Reply:
@supriyagoel The "conditioning_component" was an old schema element before it has separate boilers, chillers, pumps, etc. This rule is not mandatory so was not updated since I first created the PR.

Reply:
This is non-mandatory rule so we don't need to implement the logic, just determine whether it is applicable. 
That said, since projects often have boilers in the baseline, I think it may be worthwhile to make this rule mandatory. 
The RDS currently refers to "conditioning_component" that I don't think exists in the current schema. So, if we decide to implement it, the logic would be to check that each boiler in the B-RMR that references a fluid loop of type "heating" is natural draft.